src/JsonSchema/TypeFactory.php

```diff
@@ -27,6 +27,19 @@
  */
 final class TypeFactory implements TypeFactoryInterface
 {
+    /**
+     * This constant is to be provided as serializer context key to conditionally enable types to be generated in
+     * a format that is compatible with OpenAPI specifications **PREVIOUS** to 3.0.
+     *
+     * Without this flag being set, the generated format will only be compatible with Swagger 3.0 or newer.
+     *
+     * Once support for OpenAPI < 3.0 is gone, this constant **WILL BE REMOVED**
+     *
+     * @internal Once support for OpenAPI < 3.0 is gone, this constant **WILL BE REMOVED** - do not rely on
+     *           it in downstream projects!
+     */
+    public const CONTEXT_SERIALIZATION_FORMAT_OPENAPI_PRE_V3_0 = self::class.'::CONTEXT_SERIALIZATION_FORMAT_OPENAPI_PRE_V3_0';
```

Comment:
CS: PSR-12: `use` statements for traits must be at the top of the class, so this must be moved below.

Reply:
There's no `use` here: mis-placed comment?

Reply:
`use` statement is at line 43 below. The `const` has to be moved below it.

Reply:
https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-12-extended-coding-style-guide.md#42-using-traits

Reply:
Ah, I see: I forget about traits, since I pretty much banned them overall in almost all what I do :+1:

Reply:
dd017d27bb6406e91fcc3c5f4478338bd5a0572c